db/migrate/20190224103931_add_git_puppet_settings.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddGitPuppetSettings < ActiveRecord::Migration[5.2]
+  def change
+    add_column :app_settings, :puppet_init_path, :string, default: ''
```

Comment:
The defaults here should probably match the defaults when creating the app_settings instance

Reply:
You means like /Nodes, /Names and /Init?

Reply:
Done!